packages/http-signature-utils/src/utils/jwk.ts

```diff
@@ -1,16 +1,12 @@
 import { createPublicKey, generateKeyPairSync, KeyObject } from 'crypto'
-import { JWK as JoseWk } from 'jose'
 
-export interface JWK extends JoseWk {
+export interface JWK {
   kid: string
-  alg: string
-  kty: string
-  crv: string
+  alg: 'EdDSA'
+  use?: 'sig'
+  kty: 'OKP'
+  crv: 'Ed25519'
```

Comment:
This is what I meant when I said I thought we wanted to be less prescriptive within this package. But we can change that later. It is more consistent this way.

Reply:
Ah, got it.
fwiw we were already enforcing ed25519 on key parsing
https://github.com/interledger/rafiki/blob/a69b4d49f57b4a2c8763fc65b7095f696f2ed201/packages/http-signature-utils/src/utils/key.ts#L12-L17